rfc/2022/2022-08-10-The Console UI Will Use Material UI and Adopt Styling from the VA Design System While Isolating Plugin Styling.md

```diff
@@ -0,0 +1,149 @@
+
+These are preventive steps; however, we would need to conduct user research with people using assistive technologies to better understand the accessibility of the Console.
+
+## 3.a.2
+To ensure consistent appearance within the Console and when compared to other VA products, we suggest the following strategy to synchronize design assets for Material UI with the styling changes from Formation, the front-end framework used on [VA.gov](https://www.va.gov/). 
+
+Process for ongoing alignment of VADS, Material UI 
+Strategy for development of lifecycle of recommended sharing
+We suggest setting up a new repository for styling modifications to Backstage outside of either the Lighthouse instance or our own, so we can share and incorporate the styling into both products.
+
+In that repository, we will likely need to set up automation that downloads the [source code for Formation](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/tree/master/packages/formation). This automation would consult [assets like images and fonts, SASS files](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/master/packages/formation/sass/base/b-variables.scss) that set variables and synchronize the definitions to the design system used in Backstage. This automation could be based on the releases of [Formation](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/releases?q=Formation&expanded=true).
+
+Both the Lighthouse and the Console UI projects will then need to incorporate the changes automatically compiled, and likely manually reviewed and tweaked to this new repository.
+
+## Component alignment
+We completed an [audit of components used in the VA Design System and compared the components of both systems](https://vfs.atlassian.net/wiki/spaces/CUI/pages/2276196618/VADS+Backstage+Material+UI+Comparison+Summary). This audit revealed that the large majority of components have comparable functionality between the two systems (Material UI and VA Design System). 
+
+In addition, [we explored the technical challenges associated with how to import and use individual components from VA Design System in the Console](https://vfs.atlassian.net/wiki/spaces/CUI/pages/2256306184/Draft+Explore+replacing+components+in+Console+UI+with+Custom+Components). We concluded that overhauling the entire Console to use a set of custom components from the VA Design system presents a lot of issues. The work to implement and maintain is quite high and will likely result in more error-prone code (Backstage provided components from Material UI already handle many API interactions for us). Drifting further from Backstage/Material UI and their adopters community comes with costs as well, making updates more challenging, and not being able to benefit from community improvements to the integrated components.
```

Comment:
> We concluded that overhauling the entire Console to use a set of custom components from the VA Design system presents a lot of issues.

Can you list those issues? I see that the Confluence draft mentions that using the VA Design System instead of Material UI would involve some work - is there a demo branch or repo where we can see what that entails?

Reply:
Rolled comment into document as it wasn't directly addressed.